testing/run_tests.py

```diff
@@ -281,6 +285,8 @@ def main():
   parser.add_argument('--android-variant', dest='android_variant', action='store',
       default='android_debug_unopt',
       help='The engine build variant to run java tests for')
+  parser.add_argument('--no-verbose', dest='no_verbose', action='store_true',
```

Comment:
nit: instead of `--no-verbose`, can we call this something like `--quiet-dart-snapshot` or something?  I'd also be ok with defaulting it to True - I've yet to see that output be helpful.

Reply:
Or just `--verbose` (off by default).

Reply:
Renamed the flag to `--verbose-dart-snapshot` so by default (without giving `--verbose-dart-snapshot`) we won't show those logs.

Reply:
...Github didn't load Chinmay's comment while I was replying and modifying the code. I'll merge as is, and probably change the flag name to `--verbose` in the future if we find more verbose logging that needs to be hidden.